layouts/shortcodes/dbm-postgres-agent-config-examples.md

```diff
@@ -76,17 +99,24 @@ instances:
 ### Monitoring relation metrics for multiple logical databases
 In order to collect relation metrics (such as `postgresql.seq_scans`, `postgresql.dead_rows`, `postgresql.index_rows_read`, and `postgresql.table_size`), the Agent must be configured to connect to each logical database (by default, the Agent only connects to the `postgres` database).
 
-Specify a single "DBM" instance to collect DBM telemetry from all databases. Additionally, specify all logical databases the Agent must connect to. It is important to only have `dbm: true` on one configuration instance per host to prevent duplication of metrics.
+Specify a single "DBM" instance to collect DBM telemetry from all databases. Use the `database_autodiscovery` option to avoid specifying each logical database.
 ```yaml
 init_config:
 instances:
   # This instance is the "DBM" instance. It will connect to the
```

Comment:
there's an leftover `the` here and it reads as
> It will connect to the all logical databases

Reply:
ty!